app/demo-geant-integration/GlobalSetup.cc

```diff
@@ -43,38 +43,51 @@ GlobalSetup::GlobalSetup()
         auto& cmd = messenger_->DeclareProperty("maxNumTracks",
                                                 options_->max_num_tracks);
         cmd.SetGuidance("Set the maximum number of track slots");
-        cmd.SetDefaultValue(celeritas::Device::num_devices() > 0 ? "524288"
-                                                                 : "64");
+        if (celeritas::Device::num_devices() > 0)
+        {
+            cmd.SetDefaultValue("524288");
+            options_->max_num_events = 524288;
+        }
+        else
+        {
+            cmd.SetDefaultValue("64");
+            options_->max_num_events = 64;
+        }
```

Comment:
I think it's still ok to remove `SetDefaultValue`! I just wanted it to be replaced with *setting* the default value like the original code intended. If we still do want to set the default, maybe we could use `to_string` to simplify the code and avoid duplicating the values?
```suggestion
        options_->max_num_events = celeritas::Device::num_devices() > 0 ? 524288 : 64;
        cmd.SetDefaultValue(std::to_string(options_->max_num_events));
```

Reply:
Now that the default values are being set here as well, I thought there is some (documentation) value in keeping the calls to SetDefaultValue(). I keep the to_string() option for now.

Reply:
Perfect!